src/Module/Home.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Core\Hook;
+use Friendica\Core\L10n;
+use Friendica\Core\Renderer;
+
+/**
+ * Home module - Landing page of the current node
+ */
+class Home extends BaseModule
+{
+	public static function init()
```

Comment:
No `post()` method, you can merge `init()` and `content()`.

Reply:
But what about the hook call in the init? At least startpage is using this hook.

Reply:
Keep the hook, ditch `init()`.